### PR TITLE
git*: add pt_BR translation

### PR DESCRIPTION
**Pull request by andrealmeid**
Hello there :)

This is a first PR to create some basic git translation to Brazilian Portuguese. However, I found this translation quirk since in Brazil is very common to use neologisms when talking about git (e.g. "Eu vou **commitar**", "Precisa dar um **push**", ...). I believe the translation should keep those terms in English when applicable. Feedbacks? 

Thanks!

(cc @schneiderl)

---

- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by navarroaxel — APPROVED** [review]

File: pages.pt_BR/common/git-add.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# git add
+
+> Adiciona arquivos modificados no index
```
Comment (schneiderl) [inline pages.pt_BR/common/git-add.md:3]: Acho que o termo "staging area" ou "área de preparação" aqui fica mais próximo do contexto do Git. Isso para todos os exemplos dessa PR. 
```suggestion
> Adiciona arquivos modificados área de preparação
```

File: pages.pt_BR/common/git-remote.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# git remote
+
+> Gerencia repositórios monitorados ("remotes").
```
Comment (schneiderl) [inline pages.pt_BR/common/git-remote.md:3]: Nice!

Comment (schneiderl) [reply pages.pt_BR/common/git-remote.md:3]: Gostei disso: usar o termo em inglês à partir de uma descrição dele no início.

File: pages.pt_BR/common/git.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# git
+
+> Sistema de versionamento distruído
```
Comment (schneiderl) [inline pages.pt_BR/common/git.md:3]:
```suggestion
> Sistema de versionamento distribuído
```

**Review by schneiderl — CHANGES_REQUESTED** [review]
Hi, 
I've left a couple of minor comments for you to take a look at. Thank you a lot for your PR, they are very welcome. Nice job!

File: pages.pt_BR/common/git-add.md (hunk 1 above)
Comment (andrealmeid) [reply pages.pt_BR/common/git-add.md:3]: Feito

File: pages.pt_BR/common/git-remote.md (hunk 2 above)
Comment (andrealmeid) [reply pages.pt_BR/common/git-remote.md:3]: :)

File: pages.pt_BR/common/git.md (hunk 3 above)
Comment (andrealmeid) [reply pages.pt_BR/common/git.md:3]: Feito

**Comment by andrealmeid** [comment]
> Hi,
> I've left a couple of minor comments for you to take a look at. Thank you a lot for your PR, they are very welcome. Nice job!

@schneiderl please, have a look :)

Also, for `git-commit.md`, do you think we can keep "commit" as it is? I don't think there will be a nice translation for that.

**Comment by schneiderl** [comment]
@andrealmeid I believe you are right, there wouldn't be a nice translation for that. And also I believe that "commit" even being an English word is already well known in this context.

File: pages.pt_BR/common/git-add.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# git add
+
+> Adiciona arquivos modificados na área de preparação
+> Mais informações: <https://git-scm.com/docs/git-add>.
+
+- Adiciona um arquivo no index:
```
Comment (schneiderl) [inline pages.pt_BR/common/git-add.md:6]: Só faltou essa parte aqui. Ela passou por mim na primeira vez que estava revisando. 
```suggestion
- Adiciona um arquivo na área de preparação:
```

**Review by schneiderl — CHANGES_REQUESTED** [review]

File: pages.pt_BR/common/git-add.md (hunk 4 above)
Comment (andrealmeid) [reply pages.pt_BR/common/git-add.md]: done!

**Review by schneiderl — APPROVED** [review]
Nice! Thank you for the PR.

**Pull request merged**